onnxruntime/core/session/inference_session.cc

```diff
@@ -1151,18 +1151,19 @@ Status PartitionOrtFormatModel(onnxruntime::Graph& graph,
 
   return Status::OK();
 }
-
 #if !defined(ORT_MINIMAL_BUILD) || defined(ORT_ENABLE_RUNTIME_OPTIMIZATION_IN_MINIMAL_BUILD)
-Status ReplaySavedRuntimeOptimizations(
-    onnxruntime::Graph& graph, const logging::Logger& logger, const SessionOptions& session_options) {
+Status ApplyOrtFormatModelRuntimeOptimizations(
+    onnxruntime::Graph& graph, const logging::Logger& logger, const SessionOptions& session_options,
+    const IExecutionProvider& cpu_ep) {
   bool modified = false;
 
   for (int level = static_cast<int>(TransformerLevel::Level2);
        level <= static_cast<int>(session_options.graph_optimization_level);
        ++level) {
     // don't need to specify optimizers to disable, optimizers will process whatever runtime optimizations there are
```

Comment:
>don't need to specify optimizers to disable [](http://example.com/codeflow?start=7&length=43)

is this still true? this comment was written with replaying runtime optimizations in mind